### PR TITLE
[TTAHUB-836] Print view of goals on goals & objectives tab

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

View a printable version of the info on the goals and objectives tab. As per group discussion, we are starting with an "all or nothing" approach.

## How to test

View a recipient's goals, click "preview and print." Use the print to PDF button to generate a print preview and see how it looks.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-836

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/RecipientRecord/pages/PrintGoals.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  const [goals, setGoals] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [, setError] = useState('');
+
+  useEffect(() => {
+    const sortConfig = location.state && location.state.selectedGoals
+      ? location.state.selectedGoals
+      : [];
+
+    async function fetchGoals() {
```
Comment (thewatermethod) [inline frontend/src/pages/RecipientRecord/pages/PrintGoals.js:20]: This fetchGoals function would be replaced with a function that took an array of selected goal ids from the location state and passed those as request parameters to the backend

**Review by AdamAdHocTeam — COMMENTED** [review]
If the box for break-inside was also around the goal number would it prevent this type of break?

image

**Comment by thewatermethod** [comment]
Yeah, that's what happens if you avoid breaking inside each goal, since the goal contains the objectives, etc

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good. 
:printer:

**Review by jasalisbury — APPROVED** [review]
LGTM! Only question is on where we want to avoid breaks

File: frontend/src/pages/RecipientRecord/pages/components/PrintableObjective.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { STATUSES } from '../../../../components/GoalsTable/StatusDropdown';
+
+export default function PrintableObjective({ objective }) {
+  const key = objective.status || 'Needs Status';
+  const { icon } = STATUSES[key];
+
+  const rowClass = `ttahub-printable-objective ${ROW_CLASS}`;
+
+  return (
+    <div className="border-1px margin-y-1">
```
[the remark this answers is not in the document]
Comment (thewatermethod) [reply frontend/src/pages/RecipientRecord/pages/components/PrintableObjective.js:13]: That's a sensible question, I will bring this up at UI review tomorrow. The only issue there is that there could be a long gap between objective boxes if one has long text

**Pull request merged**